tests/Unit/Booking/BookingUnitTest.php

```diff
@@ -14,89 +12,18 @@ class BookingUnitTest extends TestCase
     use RefreshDatabase;
 
     protected $position;
+    protected $booking;
 
     protected function setUp(): void
     {
         parent::setUp();
 
-        $this->booking = factory(Booking::class)->create();
+        $this->booking = factory(Booking::class)->create(['user_id' => 1234567]);
     }
 
     /** @test */
     public function itReturnsThePositionRelation()
     {
         $this->assertInstanceOf(Position::class, $this->booking->position);
     }
-
```

Comment:
Should also add a test here - `itReturnsTheUserRelation`

Reply:
yep will add